.gitignore

```diff
@@ -1 +1 @@
-poetry.lock
+uv.lock
```

Comment:
I'm not sure we should ignore `uv.lock`; it appears to be more or less conventional to include it in the commits, so that tests run against a predictable set of dependencies. I'm not dead set on this, but I've been doing it for my libraries and it's been pretty painless.

Reply:
Sure, seems sensible enough to me 👍